src/style.css

```diff
@@ -1 +1,69 @@
 /* Styles go here */
+* {
+  padding: 0;
+  margin: 0;
+  font-family: "Roboto", sans-serif;
```

Comment:
add font-family style to body or html selector.

do not forget to add more fallback fonts
![image](https://github.com/mate-academy/layout_moyo-header/assets/22824431/ad2b02ad-02a3-4658-9606-3b26044de305)